src/Tests/Microsoft.DotNet.ApiCompatibility.Tests/SuppressionTests/SuppressionEngineTests.cs

```diff
@@ -0,0 +1,183 @@
+            engine.AddSuppression("CP0001", "T:A.B");
+
+            Assert.True(engine.IsErrorSuppressed("CP0001", "T:A.B", "ref/net6.0/myLib.dll", "lib/net6.0/myLib.dll"));
+        }
+    }
+
+    public class TestSuppressionEngine : SuppressionEngine
+    {
+        private MemoryStream _stream;
+        private StreamWriter _writer;
+        public readonly string sampleSuppressionFile = @"<?xml version=""1.0""?>
+<ArrayOfSuppression xmlns:xsi=""http://www.w3.org/2001/XMLSchema-instance"" xmlns:xsd=""http://www.w3.org/2001/XMLSchema"">
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:A.B</Target>
+    <Left>ref/netstandard2.0/tempValidation.dll</Left>
+    <Right>lib/net6.0/tempValidation.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:tempValidation.Class1.Bar(System.Int32)</Target>
+    <Left>ref/netstandard2.0/tempValidation.dll</Left>
+    <Right>lib/net6.0/tempValidation.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:tempValidation.Class1.SomeOtherGenericMethod``1(``0)</Target>
+    <Left>ref/netstandard2.0/tempValidation.dll</Left>
+    <Right>lib/net6.0/tempValidation.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:tempValidation.Class1.SomeNewBreakingChange</Target>
+    <Left>ref/netstandard2.0/tempValidation.dll</Left>
+    <Right>lib/net6.0/tempValidation.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:tempValidation.SomeGenericType`1</Target>
+    <Left>ref/netstandard2.0/tempValidation.dll</Left>
+    <Right>lib/net6.0/tempValidation.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:A</Target>
+    <Left>[tempvalidation.1.0.0]ref/netstandard1.3/tempValidation.dll</Left>
+    <Right>[tempValidation.2.0.0]ref/netstandard1.3/tempValidation.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:tempValidation.Class1</Target>
+    <Left>[tempvalidation.1.0.0]ref/netstandard1.3/tempValidation.dll</Left>
+    <Right>[tempValidation.2.0.0]ref/netstandard1.3/tempValidation.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:A</Target>
+    <Left>[tempvalidation.1.0.0]lib/netstandard1.3/tempValidation.dll</Left>
+    <Right>[tempValidation.2.0.0]lib/netstandard1.3/tempValidation.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:tempValidation.Class1</Target>
+    <Left>[tempvalidation.1.0.0]lib/netstandard1.3/tempValidation.dll</Left>
+    <Right>[tempValidation.2.0.0]lib/netstandard1.3/tempValidation.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV004</DiagnosticId>
+    <Target>.NETFramework,Version=v4.8</Target>
```

Comment:
do we support the short folder name like "net48" as well ?

Reply:
Good call I think we should support that.

Reply:
I'm going to open an issue for this. Given that the package validation errors contain the extended folder name, I think it is fine to not support the short form at the moment. Supporting it involves parsing and translating from one form to the other, so I don't think is worth the effort at the moment.